services/brig/src/Brig/API/Public.hs

```diff
@@ -1063,12 +1065,47 @@ activate (Public.Activate tgt code dryrun)
     respond (Just ident) first = ActivationResp $ Public.ActivationResponse ident first
     respond Nothing _ = ActivationRespSuccessNoIdent
 
--- Verification
-
-sendVerificationCode :: (Handler r) ()
-sendVerificationCode =
-  case Public.TeamFeatureSndFPasswordChallengeNotImplemented of
-    _ -> pure ()
+sendVerificationCode :: Public.SendVerificationCode -> (Handler r) ()
+sendVerificationCode req = do
+  let action = Public.svcAction req
+  -- TODO(leif): guard as long as not all of the actions are implemented
+  -- remove when all done
+  case action of
+    Public.GenerateScimToken -> pure ()
+    Public.Login -> do
+      let email = Public.svcEmail req
+      mbUserId <- lift $ UserKey.lookupKey $ UserKey.userEmailKey email
+      mbAccount <- lift $ join <$> Data.lookupAccount `traverse` mbUserId
+      featureStatus <- getFeatureStatus mbAccount
+      case (mbUserId, featureStatus) of
+        (Just userId, Public.TeamFeatureEnabled) -> do
+          gen <- Code.mk6DigitGen $ Code.ForEmail email
```

Comment:
Same here as in the test: it's weird that we'd have to create a full `gen` just to get the key.  But if it was like that before, maybe we shouldn't touch it.

Reply:
I'd agree, but we need the `gen` further down.